src/common/AndroidUtils.ts

```diff
@@ -804,6 +847,36 @@ export class AndroidSDKUtils {
         }
     }
 
+    // The user can provide us with emulator name as an ID (Pixel_XL) or as display name (Pixel XL).
+    // This method can be used to resolve a display name back to an id since emulator commands
+    // work with IDs not display names.
+    private static resolveEmulatorImage(
+        emulatorName: string
+    ): string | undefined {
+        const emulatorDisplayName = emulatorName.replace(/[_-]/gi, ' ').trim(); // eg. Pixel_XL --> Pixel XL, tv-emulator --> tv emulator
+
+        try {
+            const stdout = AndroidSDKUtils.executeCommand(
+                AndroidSDKUtils.getEmulatorCommand() + ' ' + '-list-avds'
+            );
+            const listOfAVDs = stdout.toString().split(os.EOL);
```

Comment:
If I recall we had tried splitting the command-line tools output like this before, only to find out that it's always `\n`, even on Windows (where `os.EOL` would be `\r\n`).  Have we verified this works on Windows?

Reply:
I haven't tested on Windows yet. I'm installing VMWare right now and was told by Techforce it's going to take few hours. I can verify it later.

Reply:
on windows it is actually `\r\n`

Reply:
@sfdctaka FYI you won't be able to test end-to-end in VMWare (or any other virtualization solution) b/c you cannot start an AVD inside another VM. But you can test things until right before launching an AVD.